### PR TITLE
V3

### PR DESCRIPTION
**Pull request by copy-ninja1**
Migrated from vue version 2 to vue version 3

### CODE REVIEW

**Comment by iamraphson** [comment]
@copy-ninja1 Thanks a lot, i will review it. Quick question, will this be back compatible? i mean,  will this work for vue 2.x?

**Comment by copy-ninja1** [comment]
> @copy-ninja1 Thanks a lot, i will review it. Quick question, will this be back compatible? i mean, will this work for vue 2.x?

I didn't bundle vue3 with this package, so it's not backward compatible.

**Comment by copy-ninja1** [comment]
@iamraphson am waiting for your feedback on this **PR**

**Comment by iamraphson** [comment]
@copy-ninja1 i will test this PR over the long break

File: lib/paystack.vue [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            );
+        };
+
+        let payWithPaystack = () => {
+            scriptLoaded.value &&
+        scriptLoaded.value.then(() => {
+            const paystackOptions = {
+                key: props.paystackkey,
+                email: props.email,
+                firstname: props.firstname,
```
[the remark this answers is not in the document]
Comment (copy-ninja1) [reply lib/paystack.vue:151]: Props are not reactive when destructured 
https://github.com/vuejs/composition-api/issues/156

[the remark this answers is not in the document]
Comment (dammy001) [reply lib/paystack.vue:154]: You can pass the props to toRefs to make it reactive.

const { email } = toRefs(props)

**Comment by iamraphson** [comment]
@copy-ninja1 can you fix the conflicts?

**Comment by copy-ninja1** [comment]
> @copy-ninja1 can you fix the conflicts?

File: lib/paystack.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (copy-ninja1) [reply lib/paystack.vue]: > 

The props have been destructured.

**Comment by copy-ninja1** [comment]
> @copy-ninja1 can you fix the conflicts?

All conflicts have been resolved

**Review by dammy001 — APPROVED** [review]

**Review by iamraphson — APPROVED** [review]
i will merge it now but i will not publish it on NPM until i am done testing. Thanks for the conttribution.

**Pull request merged**